src/Raven.Studio/typescript/viewmodels/database/tasks/editOlapEtlTask.ts

```diff
@@ -656,8 +656,10 @@ class editOlapEtlTask extends viewModelBase {
 
         if (transformation.isNew()) {
             const newTransformationItem = new ongoingTaskOlapEtlTransformationModel(transformation.toDto(), false, false);
-            newTransformationItem.name(this.findNameForNewTransformation());
+            
+            newTransformationItem.name(transformation.name() || this.findNameForNewTransformation());
```

Comment:
do we need that? Shouldn't we add validation that name is required?

Reply:
Don't we want to give users **the option** to leave the script name blank and assign 'Script_x' for the user ?

Reply:
no - we fill that initially and then require name

Reply:
ok done

Reply:
done